jsonschema/schema_test.go

```diff
@@ -131,6 +131,16 @@ func (s *SchemaSuite) TestDefaults(c *C) {
 				},
 			},
 		},
+		{
+			schema: `{
```

Comment:
It tests that if there is an empty map in the document, it gets unmarshaled as an empty map, not nil. Without my change it is unmarshaled as nil and so this test case is crashing.